lang/po/new/sv.po

```diff
@@ -1,1772 +1,1758 @@
+msgstr "\xe1ndring utf\xefrd!"
+
+# MSG_CORRECTLY c=20
+#: ultralcd.cpp:2562
+msgid "Changed correctly?"
+msgstr "\xe1ndring korrekt?"
+
+# MSG_SELFTEST_CHECK_BED c=20
+#: messages.c:94
+msgid "Checking bed"
+msgstr "Kontroll b\xe1dd"
+
+# MSG_SELFTEST_CHECK_ENDSTOPS c=20
+#: ultralcd.cpp:8481
+msgid "Checking endstops"
+msgstr "Kontroll \xe1ndl\xe1gen"
+
+# MSG_SELFTEST_CHECK_HOTEND c=20
+#: ultralcd.cpp:8487
+msgid "Checking hotend"
+msgstr "Kontroll hotend"
+
+# MSG_SELFTEST_CHECK_FSENSOR c=20
+#: messages.c:95
+msgid "Checking sensors"
+msgstr "Kontroll sensorer"
+
+# MSG_CHECKING_X c=20
+#: messages.c:23
+msgid "Checking X axis"
+msgstr "Kontroll X-axel"
+
+# MSG_CHECKING_Y c=20
+#: messages.c:24
+msgid "Checking Y axis"
+msgstr "Kontroll Y-axel"
+
+# MSG_SELFTEST_CHECK_Z c=20
+#: ultralcd.cpp:8484
+msgid "Checking Z axis"
+msgstr "Kontroll Z-axel"
+
+# MSG_CHOOSE_EXTRUDER c=20
+#: messages.c:54
+msgid "Choose extruder:"
+msgstr "V\xe1lj extruder:"
+
+# MSG_CHOOSE_FILAMENT c=20
+#: messages.c:55
+msgid "Choose filament:"
+msgstr "V\xe1lj filament:"
+
+# MSG_FILAMENT c=17
+#: messages.c:35
+msgid "Filament"
+msgstr "Mata ut filament"
+
+# MSG_WIZARD_XYZ_CAL c=20 r=8
+#: ultralcd.cpp:4951
+msgid "I will run xyz calibration now. It will take approx. 12 mins."
+msgstr "Jag kommer att utf\xefra en xyz-kalibrering nu. Det kommer att ta ca. 12 min."
+
+# MSG_WIZARD_Z_CAL c=20 r=8
+#: ultralcd.cpp:4959
+msgid "I will run z calibration now."
+msgstr "Jag kommer att utf\xefra z-kalibrering nu."
+
+# MSG_WATCH c=18
+#: messages.c:116
+msgid "Info screen"
+msgstr "Infosk\xe1rm"
+
+# MSG_INSERT_FILAMENT c=20
+#: ultralcd.cpp:2482
+msgid "Insert filament"
+msgstr "S\xe1tt i filament"
+
+# MSG_FILAMENT_LOADED c=20 r=2
+#: messages.c:38
+msgid "Is filament loaded?"
+msgstr "\xe1r filament isatt?"
+
+# MSG_STEEL_SHEET_CHECK c=20 r=2
+#: messages.c:109
+msgid "Is steel sheet on heatbed?"
+msgstr "Ligger metallskiva pa v\xe1rmeb\xe1dden?"
+
+# MSG_LAST_PRINT_FAILURES c=20
+#: messages.c:57
+msgid "Last print failures"
+msgstr "Senaste utskriftsfel"
+
+# MSG_WIZARD_WELCOME_SHIPPING c=20 r=16
+#: messages.c:122
+msgid "Hi, I am your Original Prusa i3 printer. I will guide you through a short setup process, in which the Z-axis will be calibrated. Then, you will be ready to print."
+msgstr "Hej, jag \xe1r din Prusa i3-skrivare. Jag guidar dig genom en kort installationsprocess, d\xe1r Z-axeln kommer att kalibreras. Sedan \xe1r du redo att skriva ut."
+
+# MSG_ADDITIONAL_SHEETS c=20 r=9
+#: ultralcd.cpp:5029
+msgid "If you have additional steel sheets, calibrate their presets in Settings - HW Setup - Steel sheets."
+msgstr "Om du har ytterligare metallskivor, kalibrera deras f\xefrinst\xe1llningar i Inst\xe1llningar - HW inst\xe1llning - Metallskiva."
+
+# MSG_LAST_PRINT c=18
+#: messages.c:56
+msgid "Last print"
+msgstr "Senaste utskrift"
+
+# MSG_SELFTEST_EXTRUDER_FAN c=20
+#: messages.c:89
+msgid "Left hotend fan?"
+msgstr "V\xe1nst hotend fl\xe1kt?"
```

Comment:
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.